.github/workflows/update_docker_images.yml

```diff
@@ -30,10 +30,14 @@ jobs:
       - uses: actions/checkout@v2
       - name: Build new Docker image
         run: bash "tests/scripts/execute_docker_test_env.sh" "${{ matrix.r-version }}" no_tests
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Why does building a Docker image need a GitHub token? Is download included in this step?

Reply:
The docker image builds installs R and several R packages. Some package are downloaded from Github. I recently ran into rate limits when installing the R packages, and need to forward the Github token in order to avoid those rate limits.

Reply:
Accepted.